server/execution/api/action.py

```diff
@@ -160,9 +160,12 @@ def get_perform_action_result():
             if not res.consumable:
                 res.increase()
 
+        patient_conditions = patient.activity_diagram.current.get_conditions(result_keys)
+        performed_action.result = patient_conditions
+
         return {
             "patient": patient.to_dict(),
-            "conditions": patient.activity_diagram.current.get_conditions(result_keys)
+            "conditions": patient_conditions
```

Comment:
Are these conditions redundant here? Since we're already returning the Patient object that contains the performed actions with the conditions